app/assets/javascripts/image_uploader.js

```diff
@@ -345,7 +345,8 @@ window.ST.imageUploader = function(listings, opts) {
       // Enable image resizing, except for Android and Opera,
       // which actually support image resizing, but fail to
       // send Blob objects via XHR requests:
-      disableImageResize: /Android(?!.*Chrome)|Opera/.test(window.navigator && navigator.userAgent)
+      disableImageResize: /Android(?!.*Chrome)|Opera/.test(window.navigator && navigator.userAgent),
+      maxFileSize: 10000000
```

Comment:
This is a bit messy. It's now duplicate data. I don't think there's a way out of it other than turning this file into a .js.erb and read the value from `APP_CONFIG.max_image_filesize` and that would bring lots of merge conflicts and pain to keep up-to-date with upstream's Sharetribe :see_no_evil: . I honestly prefer the duplication in this case.